pulumi/python/Pipfile

```diff
@@ -4,29 +4,25 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-awscli = "~=1.25.35"
```

Comment:
**annotation**:
The AWS cli has some pretty strict pins on its dependencies.  PyYaml specifically.  The advice should be to install the v2 aws CLI tool separately.